### ISSUE TITLE
No state to 0 update when changed via service

### ISSUE THREAD

**Issue by pos-ei-don** [htmltiger/numberbox-card#104]
When i fire this:

```
service: number.set_value
data:
  value: 0
target:
  entity_id: number.setpoint

```

my configuration is
```
  - type: custom:numberbox-card
    name: setpoint
    border: false
    entity: number.setpoint
    icon: mdi:transmission-tower
    min: -10000
    max: 12000
    step: 100
    unit: W
    speed: 160

```
the entity itself correctly shows 0, but this plugin still stays on the old value.
Wen setting it to 100, it works!!! Seems to be a problem with 0 not beeing allowed with this min/max/steps settings?

**Comment by htmltiger** [comment]
try adding `refresh: 1`

```
  - type: custom:numberbox-card
    name: setpoint
    border: false
    entity: number.setpoint
    icon: mdi:transmission-tower
    min: -10000
    max: 12000
    step: 100
    unit: W
    speed: 160
    refresh: 1
```

**Comment by htmltiger** [comment]
this should be fixed in 4.17
